### ISSUE TITLE
broswer-test-components ERR_NOT_RESOLVED

### ISSUE THREAD

**Issue by raoufswe** [justeattakeaway/pie#763]
when you make a change to the Pie docs site, the `browser-test-components`  job fails and throws the following error:

![image](https://github.com/justeattakeaway/pie/assets/28605803/3c8f6cb4-724f-4810-99bf-022d7718a9bf)

I closed my PR https://github.com/justeattakeaway/pie/pull/760 and created a new one but that didn't help. 

the error can be reproduced locally when you run the Pie Docs visual tests without running the dev server.

**Comment by maledr5** [comment]
I think the issue is because the `browser-test-components` should not be running `pie-docs` tests. The docs are actually deployed in a subsequent job. 
Maybe we can use a similar filter used in `Get change type` `--filter='{./packages/components/*}[origin/main]'`, to replace this one:

https://github.com/justeattakeaway/pie/blob/2204e0a38b1117b52fda7a74dd0996660aaf0fd7/.github/workflows/ci.yml#L214

@siggerzz @JoshuaNg2332 let me know your thoughts :)

**Comment by jamieomaguire** [comment]
> I think the issue is because the `browser-test-components` should not be running `pie-docs` tests. The docs are actually deployed in a subsequent job. Maybe we can use a similar filter used in `Get change type` `--filter='{./packages/components/*}[origin/main]'`, to replace this one:
> 
> https://github.com/justeattakeaway/pie/blob/2204e0a38b1117b52fda7a74dd0996660aaf0fd7/.github/workflows/ci.yml#L214
> 
> @siggerzz @JoshuaNg2332 let me know your thoughts :)

@maledr5 yeah it seems strange that the docs tests are running. The filter argument looks like it should be limiting the projects only to those in the components directory 🤔
